src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
-'use strict';
+"use strict";
 
-// write code here
+document.querySelectorAll(".tree li").forEach((li) => {
+  const span = document.createElement("span");
+
+  li.prepend(span);
+  span.appendChild(span.nextSibling);
+});
+
+document.querySelector(".tree").addEventListener("click", function (e) {
+  if (e.target.tagName !== "SPAN") {
+    return;
+  }
+
+  const childrenContainer = e.target.parentNode.querySelector("ul");
```

Comment:
The code will not do anything if there is no 'ul' element. Consider adding a condition to check if the clicked span has children.